tests/tools.vitruv.applications.pcmjava.tests/src/tools/vitruv/applications/pcmjava/tests/pcm2java/javahelper/JavaStdLibCompilationUnitHelper.xtend

```diff
@@ -0,0 +1,32 @@
+package tools.vitruv.applications.pcmjava.tests.pcm2java.javahelper
+
+import org.eclipse.emf.ecore.util.EcoreUtil
+import org.emftext.language.java.containers.CompilationUnit
+import tools.vitruv.framework.views.View
+
+import static tools.vitruv.applications.pcmjava.tests.pcm2java.JavaQueryUtil.*
+
+import static extension edu.kit.ipd.sdq.commons.util.java.lang.IterableUtil.claimOne
+
+/*
+ * This class is a workaround to save the time-consuming programmatic creation
+ * of classes from the Java standard library. The required CompilationUnit is 
+ * already present in the model to be tested (getJavaCompilationUnits) and is 
+ * then copied instead of creating it from scratch.
+ */
+class JavaStdLibCompilationUnitHelper {
```

Comment:
Maybe as a solution to also fix the final not working test:
Instead of requiring this workaround, would it be possible to ignore compilation units / classifiers of the JavaStdLib in the validation?